psrecord/main.py

```diff
@@ -90,6 +90,10 @@ def main():
                              'in a slower maximum sampling rate).',
                         action='store_true')
 
+    parser.add_argument('--use-unix-ts',
```

Comment:
Even though it would be a little more verbose, I think I'd prefer ``--include-timestamp`` and make it so that it simply adds a column (e.g. at the start) rather than replace the relative time.